mjolnir/mjolnir.py

```diff
@@ -0,0 +1,111 @@
+"""
+MIT License
+
+Copyright (c) 2020-2021 Jojo#7711
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+import asyncio
+import logging
+import random
+import typing
+from contextlib import suppress
+
+import discord
+from redbot.core import Config, commands
+from redbot.core.bot import Red
+from redbot.core.utils.chat_formatting import pagify
+
+from . import menus
+
+log = logging.getLogger("red.JojoCogs.mjolnir")
+sayings = (
+    "The hammer is strong, but so are you. Keep at it!",
+    "Mjolnir budges a bit, but remains steadfast, as should you",
+    "You've got this! I believe in you!",
+    "Don't think it even moved... why don't you try again?",
+)
+
+class Mjolnir(commands.Cog):
+    """Attempt to lift Thor's hammer!"""
+
+    __version__ = "0.1.1"
+
+    def __init__(self, bot: Red):
+        self.bot = bot
+        self.config = Config.get_conf(self, 1242351245243535476356, True)
+        self.config.register_user(lifted=0)
+
+    async def red_delete_data_for_user(self, *, requester, user_id):
+        await self.config.user_from_id(user_id).clear()
+
+    @commands.command()
+    async def lifted(self, ctx):
+        """Shows how many times you've lifted the hammer."""
+        lifted = await self.config.user(ctx.author).lifted()
+        plural = "s" if lifted != 1 else ""
+        await ctx.send(f"You have lifted Mjolnir {lifted} time{plural}")

```

Comment:
```suggestion
        await ctx.send(f"You have lifted Mjolnir {lifted} time{plural}.")

```

It's my OCD, sorry :P